### PR TITLE
Update autosuspend.py

### PR DESCRIPTION
**Pull request by das-nervt**
Got the following error:
```
# python3 /usr/bin/autosuspend.py 
Traceback (most recent call last):
  File "/usr/bin/autosuspend.py", line 686, in <module>
    main()
  File "/usr/bin/autosuspend.py", line 682, in main
    all_checks=args.all_checks)
  File "/usr/bin/autosuspend.py", line 509, in loop
    result = check.check()
  File "/usr/bin/autosuspend.py", line 236, in check
    for user, terminal, host, started in psutil.users():
ValueError: too many values to unpack (expected 4)
```
solved with this change

### CODE REVIEW

**Comment by languitar** [comment]
Do you know in which version of psutil this changed?

File: autosuspend.py [hunk 1]
```diff
@@ -272,7 +272,7 @@ def check(self):
                 if line.startswith('----'):
                     start_seen = True
 
-        if connections:
+        if len(connections) > 1:
```
Comment (languitar) [inline autosuspend.py]: Why is this necessary?

Comment (das-nervt) [reply autosuspend.py]: this Smb check is (on my machine (samba 4.7.0)) always true, because the output of 'smbstatus -b'ends with /n. Looks like Split finds minimum one line after the '----' ?

Comment (languitar) [reply autosuspend.py]: @das-nervt that is weird. Could you please provide the raw output of `smbstatus -b` in case no one is connected

Comment (languitar) [reply autosuspend.py]: Ok, I see the issue. Weird that I did not notice this on my own. I will address this separately. Could you please remove this commit from the PR as this is unrelated to the `psutils` issue.
